docs/docsite/rst/porting_guides/porting_guide_2.8.rst

```diff
@@ -125,6 +125,8 @@ Plugins
   has changed to ``%USERPROFILE%\.ansible_async``. To control this path now, either set the ``ansible_async_dir``
   variable or the ``async_dir`` value in the ``powershell`` section of the config ini.
 
+* The ``default`` callback plugin now has two additional stat counters in play recap: ``ignored`` for tasks that failed but had set ``ignore_errors: yes`` and ``rescued`` for tasks that failed initially but the rescue section was executed. Note that ``rescued`` tasks are no longer counted as ``failed`` as in Ansible 2.7 (and earlier).
```

Comment:
I'd start with the part users will see, then describe how it works:

"Play recap now counts `ignored` and `rescued` tasks as well as `changed` and `skipped` and `failed` tasks, thanks to two additional stat counters in the `default` callback plugin. Tasks that fail and have `ignore_errors: yes` set are listed as `ignored`. Tasks that fail and then execute a rescue section are listed as `rescued`. Note that `ignored` tasks are also included in the total count of `failed` tasks, but `rescued` tasks are not."

Reply:
Thanks for re-wording. Couple of notes though:
* "`ok`, `changed`, `skipped` and `failed` tasks" - if we're listing all currently existing counters, should we mention `unreachable` too?
* "Note that ignored tasks are also included in the total count of failed tasks" - this isn't true as `ignored` and `failed` are mutually exclusive. So I'd leave the last sentence as currently is in the PR (or re-word it) since it clarifies what has changed from previous versions?